shared/admin-tools/aws/migrationWaitHelper.ts

```diff
@@ -9,7 +9,7 @@ import { getItem, putItem } from './deployTableHelper';
 
 const cloudwatchClient = new CloudWatchClient({ region: 'us-east-1' });
 const env = process.env.STAGE;
-const key = 'migration-queue-is-empty';
+const key = 'migration-queue-empty';
```

Comment:
Oh my gosh I did this!

![](https://media3.giphy.com/media/v1.Y2lkPTc5MGI3NjExYWNkM3F0M2d1cHU5OW1wZXIxeDYyZTlreXR0cm44ZHkxMW1iczI4cyZlcD12MV9pbnRlcm5hbF9naWZfYnlfaWQmY3Q9Zw/3o7btUg31OCi0NXdkY/giphy.webp)

Reply:
It's easy to do! Don't sweat it